woodwork/column_accessor.py

```diff
@@ -195,3 +205,7 @@ def set_semantic_tags(self, semantic_tags):
         self._schema['semantic_tags'] = _set_semantic_tags(semantic_tags,
                                                            self.logical_type.standard_tags,
                                                            self.use_standard_tags)
+
+
+def _raise_init_warning():
```

Comment:
This name makes it sound like we're warning users and not stopping them with an Error

Reply:
Yep - I'll change. I don't know why I got tripped up on warning versus error here. I also did the same thing with the title of the pr...